github-status/handler.go

```diff
@@ -170,6 +167,38 @@ func buildPublicStatusURL(status string, statusContext string, event *sdk.Event)
 	return url
 }
 
+func formatDashboardURL(gatewayURL string, eventOwner string) (string, error) {
+	systemURL, formatErr := formatSystemURL(gatewayURL)
+	if formatErr != nil {
+		return "", fmt.Errorf("error while formatting dashboard URL: %s", formatErr.Error())
+	}
+
+	return fmt.Sprintf("%s/dashboard/%s", systemURL, eventOwner), nil
+}
+
+func formatSystemURL(gatewayURL string) (string, error) {
+	if strings.HasSuffix(gatewayURL, "/") {
+		gatewayURL = strings.TrimSuffix(gatewayURL, "/")
+	}
+	subDomain, err := getSubDomain(gatewayURL)
+	if err != nil {
+		return "", fmt.Errorf("error while geting subdomain for system URL: %s", err)
+	}
+	systemURL := strings.Replace(gatewayURL, subDomain, "system", -1)
+	return systemURL, nil
+}
+
+func getSubDomain(URL string) (string, error) {
+	parsedURL, parseErr := url.Parse(URL)
+	if parseErr != nil {
+		return "", fmt.Errorf("Unable to parse URL: %s", parseErr.Error())
+	}
+	subdomain := strings.Split(parsedURL.Host, ".")
+
+	//Host is www.world.org and subdomain would be www aka. 0th element of the slice
+	return subdomain[0], nil
+}
+
```

Comment:
Ivana I think you are right those can be added to SDK since I use it also the same way and later PR can be added to use them from `sdk.GetSubDomain` etc. If you want you can add them or I can ping me, I have tests for the functions though so I think it will be better If I add them for you?

Reply:
Thanks @martindekov 
If you can move them to SDK, then there won't be need to edit the gitlab code, if it's not merged yet.
If you're busy, I can also move them, I'm fine with both

Reply:
I think it will be easier for me to do it since I have already the tests I will just copy the code in the SDK, don't worry about it

Reply:
Great, thanks :)

Reply:
No problemos